### PR TITLE
update icon: jira (original, original-wordmark, plain, plain-wordmark) 

### PR DESCRIPTION
**Pull request by jusuchin85**
# **New icon section**

## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] The PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [ ] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-) - `not required as this is an update to an existing icon set`

## Link to prove your SVG is correct and up-to-date

https://atlassian.design/resources/logo-library/

---

Icons are updated to reflect the same design as #1044, #1045 and #1061 .

### CODE REVIEW

**Comment by jusuchin85** [comment]
- Reupload original icon design (in case of a rollback)
- Icons resized accordingly using [iloveimg](https://www.iloveimg.com/resize-image/resize-svg#resize-options,pixels).

File: icons/jira/jira-plain-wordmark.svg [hunk 1]
```diff
@@ -1 +1,22 @@
...
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Panquesito7) [inline icons/jira/jira-plain-wordmark.svg:1]: Does this icon count as plain? It has more than 1 color, which is a gradient color. I'd recommend keeping it one color only; same for the `plain` version.

Comment (jusuchin85) [reply icons/jira/jira-plain-wordmark.svg:1]: Gotcha. I'll update with a new commit.

**Comment by jusuchin85** [comment]
Done @Panquesito7

File: icons/jira/jira-plain-wordmark.svg [hunk 2]
```diff
@@ -1 +1,129 @@
...
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (Panquesito7) [inline icons/jira/jira-plain-wordmark.svg:1]: Hmm, I still see one different color in the logo. Do you need any help? Let us know. :)

Comment (jusuchin85) [reply icons/jira/jira-plain-wordmark.svg:1]: Ah really?? Lemme's double-check first and see where I missed out

**Review by Thomas-Boi — APPROVED** [review]
Looks good to me. Thank you 👍

**Pull request merged**